test/acceptance/spec/21-death-details.js

```diff
@@ -23,7 +23,6 @@ describe('Death details page', () => {
     it('a limited version is displayed in a table', () => {
       const browserText = browser.$$('table tr');
       const tableText = browser.$('table').getText();
-      console.log('text ======>\n', tableText);
```

Comment:
Shoot the reviewer!

Reply:
Haha, yeh well I think fault still lies with the mug that left debug code in the PR!